catalog/ui/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -72,6 +72,16 @@ define([
     function translateFilterToBasicMap(filter){
         var propertyValueMap = {};
         var downConversion = false;
+
+        if (filter.filters && isAnyDate(filter)) {
```

Comment:
You should try if at all possible to be explicit and avoid coercion because it's a) harder to read and understand b) more likely to break or do something you don't want it to do (because of coercion)
If you are wishing to check that it is null you should do `filter.filters === null`